### PR TITLE
Changed Position of Hamburger Icon from middle to left corner.

### PR DESCRIPTION
**Pull request by jainarunarunjain**
@deshraj @aka-jain @spyshiv Please Review the PR.

### CODE REVIEW

File: frontend/src/css/modules/main-header.scss [hunk 1]
```diff
@@ -39,6 +39,12 @@
             color: $highlight;
             background-color: rgba(0, 0, 0, 0);
         }
+        a.main-header-link {
```
Comment (aka-jain) [inline frontend/src/css/modules/main-header.scss]: @jainarunarunjain you just need to add a media query for this. Try to add with screen size less than 992px.

Comment (jainarunarunjain) [inline frontend/src/css/modules/main-header.scss]: @aka-jain Sure.

**Comment by jainarunarunjain** [comment]
@aka-jain @spyshiv Please Review.

**Comment by aka-jain** [comment]
LGTM. Merging it.

**Pull request merged**